src/plugins/discover/public/application/discover_router.tsx

```diff
@@ -6,6 +6,7 @@
  * Side Public License, v 1.
  */
 
+// eslint-disable-next-line no-restricted-imports
```

Comment:
I wonder why this file is handled differently. Should not it also be refactored to importing from  `@kbn/shared-ux-router`?

Reply:
you are right @jughosta, I had some issues initially with the discover, so that's why I have decided to keep the old router here, I will check if I can refactor it as well. Great catch :)

Reply:
@jughosta fixed :)

Reply:
Thanks!